### PR TITLE
[TTS] remove the avoidance of circular imports

### PR DESCRIPTION
**Pull request by XuesongYang**
Signed-off-by: Xuesong Yang <1646669+XuesongYang@users.noreply.github.com>

# What does this PR do ?

It is no problem to run through below after remove the line.
``` bash
In [2]: from nemo.collections.tts.models.base import SpectrogramGenerator

In [3]: spectrogram_model = SpectrogramGenerator.from_pretrained("tts_en_fastpitch")
100% [......................................................................] 187023360 / 187023360

In [4]: spectrogram_model
Out[4]:
FastPitchModel(
  (mel_loss): MelLoss()
  (pitch_loss): PitchLoss()
  (duration_loss): DurationLoss()
  (aligner): AlignmentEncoder(
    (softmax): Softmax(dim=3)
    (log_softmax): LogSoftmax(dim=3)
    (key_proj): Sequential(
      (0): ConvNorm(
        (conv): PartialConv1d(384, 768, kernel_size=(3,), stride=(1,), padding=(1,))
      )
      (1): ReLU()
      (2): ConvNorm(
        (conv): PartialConv1d(768, 80, kernel_size=(1,), stride=(1,))
```

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/tts/models/__init__.py [hunk 1]
```diff
@@ -21,17 +21,3 @@
...
-    "TwoStagesModel",
-    "GriffinLimModel",
-    "MelPsuedoInverseModel",
-    "HifiGanModel",
-    "FastPitchModel",
-    "AlignerModel",
-    "MixerTTSModel",
-    "UnivNetModel",
-    "RadTTSModel",
-]
```
Comment (blisc) [inline nemo/collections/tts/models/__init__.py:37]: Please leave this as is

Comment (XuesongYang) [reply nemo/collections/tts/models/__init__.py:37]: these lines are only used for wildcard imports, right? I did not find anywhere `import *` except the "avoid circular imports" line

Comment (blisc) [reply nemo/collections/tts/models/__init__.py:37]: Still better to have it in case anyone decides to do `import *`

Comment (XuesongYang) [reply nemo/collections/tts/models/__init__.py:37]: sounds good. Will revert it. Thanks!

**Review by blisc — APPROVED** [review]

**Pull request merged**